Gems/AWSCore/Code/Tests/ResourceMapping/AWSResourceMappingManagerTest.cpp

```diff
@@ -59,6 +59,34 @@ R"({
     "Version": "1.0.0"
 })";
 
+static constexpr const char TEST_VALID_EMPTY_ACCOUNTID_RESOURCE_MAPPING_CONFIG_FILE[] =
```

Comment:
Do we need another test case where the global AccountId field doesn't exist in the config file at all?

Reply:
discussed offline: previous PR has been updated to ensure key exists (but can be empty)